### PR TITLE
feat: Vendor feature

### PR DESCRIPTION
**Pull request by nataniadeandra**
This PR introduces several updates to the vendor feature of our project. The main additions include:

1. **Vendor Feature Tests:** Implemented backend tests for the vendor feature. These tests cover both the vendor details and the vendor photo gallery.

2. **Vendor CRUD Operations:** Added APIs for creating, reading, updating, and deleting (CRUD) vendors. Also implemented create, read, and delete (CRD) operations for photos in the gallery.

### CODE REVIEW

**Review by JohannesSetiawan — APPROVED** [review]
LGTM. Good implementation

File: revelio/settings_dev.py [hunk 1]
```diff
@@ -163,8 +163,7 @@
 # Static files (CSS, JavaScript, Images)
 # https://docs.djangoproject.com/en/5.0/howto/static-files/
 
-# STATIC_URL = 'static/'
```
Comment (catherineeangel) [inline revelio/settings_dev.py:166]: remove this is not used

**Review by carlenee — APPROVED** [review]
LGTM. Nice code very neat

**Review by catherineeangel — COMMENTED** [review]
Overall the code looks good.

File: revelio/settings_dev.py (hunk 1 above)
Comment (nataniadeandra) [reply revelio/settings_dev.py:166]: I have uncommented this part. Will be used for Django Admin styling.

**Pull request merged**